packages/perseus-editor/src/widgets/label-image-editor.tsx

```diff
@@ -28,7 +31,7 @@ type Props = {
     imageWidth: number;
     imageHeight: number;
     // The list of label markers on the question image.
-    markers: ReadonlyArray<MarkerType>;
+    markers: ReadonlyArray<LabelImageMarker & LabelImageMarkerScoringData>;
```

Comment:
Nit: I notice your creating this union in a lot of files. Can you combine them in the types object so you don't have to keep joining them in every file?

Reply:
Great suggestion! I'll set that up. :)